app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -674,6 +674,7 @@ class ChatController(args: Bundle) :
         }
 
         binding?.scrollDownButton?.setOnClickListener { binding?.messagesListView?.smoothScrollToPosition(0) }
```

Comment:
@AndyScherzinger 
smoothScrollToPosition is a bit jerky for me.
i suggest we change smoothScrollToPosition to scrollToPosition which doesn't have the automated scrolling but feels smoother UX wise..?

Reply:
Yes, just did via https://github.com/nextcloud/talk-android/pull/2741/commits/45f6e3b47426d6dd1fa6bd869adace8c3f839525 - also check whatsapp and they also do it like you suggested @mahibi 👍